doc/user/sql/types/float.md

```diff
@@ -22,7 +22,29 @@ Type               | Aliases           | Size    | Range
 
 - Materialize assumes untyped numeric literals containing decimal points are [`decimal`](../decimal); to use `float`, you must explicitly cast them as we've done below.
 
-## Details
+### Special values
+
+Floating-point numbers have three special values, as specified in IEEE 754:
+
+Value       | Aliases                    | Represents
+------------|----------------------------|-----------
+`NaN`       |                            | not a number
```

Comment:
I prefer capitalization of each `td` element that doesn't being with `code`

Reply:
Sure, done!